Microsoft.Toolkit.Uwp.SampleApp/Styles/ThemeInjector.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Foundation.Metadata;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.Styles
+{
+    public static class ThemeInjector
```

Comment:
Could be useful to generalize this in the future for multiple themes beyond light/dark as well.  Maybe we raise a GitHub issue for that later?

Reply:
Perhaps, but neither Windows or us have any other Themes.